src/components/FavoriteButton/index.js

```diff
@@ -24,21 +24,31 @@ class favoriteButton extends Component {
   }
 
   render () {
-    const schoolData = {
-      school_id: this.props.schoolInfo.id,
-      school_address: this.props.schoolInfo.address,
-      school_website: this.props.schoolInfo.website,
-      school_name: this.props.schoolInfo.name,
-      school_code: this.props.schoolInfo.dps_school_code,
-      user_id: this.props.userId
+    const { schoolInfo, commuteInfo, userId, deleteFavorite, makeFavorite } = this.props
+    let schoolData = {};
+    
+    if (commuteInfo) {
```

Comment:
had to conditionally make the schoolData object, its only needed when posting to favorites.  so when rendering the fav button on a school that is being rendered in the user profile this info isnt need but if not conditionally rendered will throw an error because commuteInfo isn't available in the profile container, only in the filter container